templates/app/controllers/spree/order_contents_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Spree
```

Comment:
Missing `frozen-string-literals` comment 🙂

Reply:
Oops! Missed that one. Fixed!